internal/api/handlers/schedulers_handler_test.go

```diff
@@ -54,7 +56,10 @@ import (
 )
 
 func TestGetAllSchedulers(t *testing.T) {
-
+	err := validations.RegisterValidations()
```

Comment:
@caiordjesus  why did you register the validations here? I think it's not necessary.